app/controllers/support_interface/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module SupportInterface
+  class UsersController < SupportInterfaceController
+    include Pagy::Backend
+
+    def index
+      @all_users ||= IdentityApi.get_users
+      @total = @all_users.size
+      @pagy, @users = pagy_array(@all_users)
+    rescue IdentityApi::ApiError,
+           Faraday::ConnectionFailed,
+           Faraday::TimeoutError,
+           ActionController::ActionControllerError => e
+      Sentry.capture_exception(e)
+      # Do not redirect back to Get An Identity
+      render "errors/internal_server_error", status: :internal_server_error
```

Comment:
Is this code doing anything? Without this, it'll just raise the error, it'll not get caught, Sentry will capture it and we'll display the 500 anyway. I guess one thing this is doing is saying "yeah, we expect this to fail, and we are being purposeful about the fact that we're displaying the 500 here for now instead of letting it just happen on its own"